app/static/audio_settings.js

```diff
@@ -0,0 +1,170 @@
+// Event listeners
+const set_checked_ids = [
+  "Music_On",
+  "Music_Off",
+  "Sound_Effects_On",
+  "Sound_Effects_Off",
+];
+const other_ids_and_their_funcs = [
+  ["activate", set_default],
+  ["on", start_audio],
+  ["off", stop_audio],
+];
+set_checked_ids.forEach((val) => {
+  add_set_checked_listener(val);
+});
+other_ids_and_their_funcs.forEach((val) => {
+  perform_func_on_click(val[0], val[1]);
+});
```

Comment:
Wait for the entire document to load before you perform this functions.
Currently it can run before the whole document loads, and it'll be buggy.

You can use `addEventListener` to wait for the page to load.